### PR TITLE
Rubocop formatting

### PR DESCRIPTION
**Pull request by orien**
Let Rubocop loose on our codebase.

robo

### CODE REVIEW

**Comment by asellitt** [comment]
death by 1K lines of change! looks good though +1

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# TODO Enable these checks and fix codes
+
+Style/ClassVars:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/DoubleNegation:
+  Enabled: false
```
Comment (orien) [inline .rubocop.yml]: Still a bunch of things to fix. Let's do that in separate PR's.

File: Gemfile [hunk 2]
```diff
@@ -1,2 +1,2 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
```
Comment (spickermann) [inline Gemfile]: AAAAHHHHHHHHH!

File: lib/active_record/locking_extensions.rb [hunk 3]
```diff
@@ -42,23 +36,21 @@ def create_ignoring_duplicates!(*args)
       end
     end
 
-    private
+  private
```
Comment (spickermann) [inline lib/active_record/locking_extensions.rb]: I like!

File: Gemfile (hunk 2 above)
Comment (orien) [inline Gemfile]: prefer the other way?

Comment (spickermann) [inline Gemfile]: Yes, I would prefer single quotes. But, please, don't start a discussion on this topic again. Just tell me what to use...

Comment (orien) [inline Gemfile]: Seems like the project is currently 50/50. It's OK to go the other way.

**Comment by spickermann** [comment]
Awesome!
